script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -72,7 +72,7 @@ The one who smashed the third-floor window and came tumbling inside is―――
 -- Page 81, Offset 30767.
 -- 「……数秘門による雷霆ですか。
 // 数秘門 is a typo here, this is supposed to be 数秘紋
-"...Gematria-based lightning. I see.
+"Numeromancy-based lightning... I see.
```

Comment:
numeromancy sounds either super dope or super dope-y and i can't decide which

Reply:
It's dope